WordPress/Sniff.php

```diff
@@ -1505,7 +1505,8 @@ protected function is_in_isset_or_empty( $stackPtr ) {
 		end( $nested_parenthesis );
 		$open_parenthesis = key( $nested_parenthesis );
 
-		return \in_array( $this->tokens[ ( $open_parenthesis - 1 ) ]['code'], array( \T_ISSET, \T_EMPTY ), true );
```

Comment:
So when looking for code-style assumptions, a good starting point is `$this->tokens[ $var +/- X ]`?

Reply:
Yes and no. doing that calculation is very common and correct when it's to set the starting point for a `findNext()`/`findPrevious()` function call.
Code style assumptions are only problematic for best practice sniffs, not code style sniffs and to bypass those assumption you need to take every crappy way of writing PHP into account, so looking for checks using `T_WHITESPACE` is another typical search entry point. This should - for code style independence - be `Tokens::$emptyTokens`.